### PR TITLE
Initial service scaffolding

### PR DESCRIPTION
**Pull request by foklepoint**

### CODE REVIEW

**Comment by foklepoint** [comment]
@MirTalpur

File: cmd/main.go [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+	Port int
+}
+
+func main() {
+	config := getServiceConfiguration()
+	logger := getLogger()
+	startServer(config, logger)
+}
+
+func startServer(config *ServiceConfiguration, logger kitlog.Logger) {
```
Comment (MirTalpur) [inline cmd/main.go]: Why not have a type for a Server with an constructor that takes in the config and logger.
The methods getServiceConfiguration, getLogger and startServer etc. can then be called on the Server instance.

**Comment by foklepoint** [comment]
Closed in favor of #2

**Comment by foklepoint** [comment]
Closed in favor of #2

**Pull request closed without merging**